phalcon/security.zep

```diff
@@ -390,15 +393,15 @@ class Security implements InjectionAwareInterface
 			/**
 			 * We always check if the value is correct in post
 			 */
-			let userToken = request->getPost(tokenKey);
+			let userToken = (string) request->getPost(tokenKey);
```

Comment:
Can you change this to:

`let userToken = request->getPost(tokenKey, "string");`

Direct casting is faster for sure but I prefer passing the token here via the sanitization/filter